test/integration/puppetlabs/services/master/master_service_test.clj

```diff
@@ -48,6 +53,22 @@
                       :headers {"Accept" "pson"}
                       :as :text})))
 
+(defn http-put
+  [url body]
+  (let [master-service-test-runtime-ssl-dir
+        (str master-service-test-runtime-dir "/ssl")]
+    (http-client/put (str "https://localhost:8140" url)
+                     {:ssl-cert (str master-service-test-runtime-ssl-dir
+                                     "/certs/localhost.pem")
+                      :ssl-key (str master-service-test-runtime-ssl-dir
+                                    "/private_keys/localhost.pem")
+                      :ssl-ca-cert (str master-service-test-runtime-ssl-dir
+                                        "/certs/ca.pem")
+                      :body body
+                      :headers {"Accept" "pson"
+                                "Content-type" "text/pson"}
```

Comment:
Yeah, I tried switching to json here and it didn't work. I'll put in a ticket about updating all our tests to use json in headers rather than pson.